### ISSUE TITLE
[kubectl web terminal] remote terminal leak.

### ISSUE THREAD

**Issue by lixd** [kubesphere/kubesphere#5023]
**Describe the Bug**
kubectl web terminal conn leak. when frontend websocket closed,remote terminal not close.

**Versions Used**
KubeSphere: v3.2.1

**Environment**
How many nodes and their hardware configuration:

CentOS 7.9 1mster 4c8g.

**How To Reproduce**
![image](https://user-images.githubusercontent.com/31980412/176982065-0fded523-5291-4674-ad2b-ee26ee83a65d.png)

1.click to open web terminal
2.close web terminal
3.repeat step1、2

use this cmd to get conn number:
```bash
pod=ks-apiserver-5ddb867984-8b5gz
containerID=$(kubectl get po $pod -n kubesphere-system -o yaml|grep "containerID: c"|sed 's/  - containerID: containerd:\/\///')
pid=$(crictl inspect $containerID | grep '"pid":'|head -2|head -1|sed -e 's/    \"pid\": //' -e 's/,//')
lsof -p $pid | wc -l

```
or just use this
```
ps -ef|grep sh
```
you will found many `sh` process.

**Expected behavior**
when the web terminal closed, the remote terminal closed too.

**Comment by wansir** [comment]
Duplicated https://github.com/kubesphere/kubesphere/issues/4917, already fixed in https://github.com/kubesphere/kubesphere/pull/4918.

**Comment by lixd** [comment]
> Duplicated #4917, already fixed in #4918.

maybe there is another issue,caseby SPDYExecutor,see [How to cancel a SPDYExecutor stream?](https://github.com/kubernetes/client-go/issues/554)
